### PR TITLE
Add .env python code

### PR DESCRIPTION
**Pull request by Hussam-Suleiman**

### CODE REVIEW

File: lib/config.py [hunk 1]
```diff
@@ -8,12 +8,44 @@
...
+BOUNCER_LISTEN_PORT_ODOO = os.environ["BOUNCER_LISTEN_PORT_ODOO"]
+BOUNCER_ODOO_DATABASE_ODOO = os.environ["BOUNCER_ODOO_DATABASE_ODOO"]
+BOUNCER_ODOO_URL_ODOO = os.environ["BOUNCER_ODOO_URL_ODOO"]
+BOUNCER_SMTP_FROM_ODOO = os.environ["BOUNCER_SMTP_FROM_ODOO"]
+BOUNCER_SMTP_PASS_ODOO = os.environ["BOUNCER_SMTP_PASS_ODOO"]
+BOUNCER_SMTP_PORT_ODOO = os.environ["BOUNCER_SMTP_PORT_ODOO"]
+BOUNCER_SMTP_SERVER_ODOO = os.environ["BOUNCER_SMTP_SERVER_ODOO"]
+BOUNCER_SMTP_SSL_ODOO = os.environ["BOUNCER_SMTP_SSL_ODOO"]
+BOUNCER_SMTP_TO_ODOO = os.environ["BOUNCER_SMTP_TO_ODOO"]
+BOUNCER_SMTP_USER_ODOO = os.environ["BOUNCER_SMTP_USER_ODOO"]
```
Comment (thomaspaulb) [inline lib/config.py:40]: We don't need this, we won't be using these variables anywhere.

File: generate-secret-code.py [hunk 2]
```diff
@@ -4,5 +4,5 @@
 
 print(
     "Copypaste this line into .env:\n"
-    "NGINX_ODOO_HOTP_SECRET={}".format(pyotp.random_base32()),
+    "BOUNCER_HOTP_SECRET_ODOO={}".format(pyotp.random_base32()),
```
Comment (thomaspaulb) [inline generate-secret-code.py:7]: It's good that you renamed all these variables, but will you also run a `sed` command on `dng`, `dngtest`, `freshfilter` and `gedeeldeweeldetest` containers for the `.env` files after this change?

Comment (Hussam-Suleiman) [reply generate-secret-code.py:7]: yes if you accept this name I will migrate it and fix .env files

File: lib/config.py (hunk 1 above)
Comment (Hussam-Suleiman) [reply lib/config.py:40]: which variables? BOUNCER_SMTP_*_ODOO?

File: generate-secret-code.py (hunk 2 above)
Comment (thomaspaulb) [reply generate-secret-code.py:7]: Okay good, and after you are done, test on their URL's, or ping me so I can test it

File: lib/config.py (hunk 1 above)
Comment (thomaspaulb) [reply lib/config.py:40]: Yes

**Review by thomaspaulb — APPROVED** [review]

**Pull request merged**